### DISCUSSION TITLE
Monitoring and Control Hardware for custom built Liquid Cooling Systems

### DISCUSSION THREAD

**Discussion by S-Essig** [liquidctl/liquidctl#504] [Ideas]
Hi everybody,
I discovered this project by trying to find a way to get the temperature of two 10k NTC Sensors of my custom water cooling system to be displayed in HWInfo. liquidctl is a very cool project and I would like to contribute a Hardware Module (maybe using RP2040 as a Microcontroller with an internal USB connection, the device could register as a COM Port). The Hardware Module could provide PWM control for Pumps / Fans (including Speed readout) with non volatile memory, several Temperature Sensors like DS18B20 and the mentioned 10k NTC as well asRGB Lighting connectors for DRGB (5V adressable) and RGB (12V PWM).

If there is an interest a Hardware Module, the specifications are of course open for discussion!

What I do know:
- PCB design
- Microcontroller programming

What I do not know:
- typical communication Interface for AIOs
- coding drivers to use the board with Windows/Linux

Are the devs interested in a HW Module for liquidctl?

**Comment by jonasmalacofilho** [comment]
Hey @S-Essig ,

That's really cool, and something I might eventually use & contribute to; but right now I really can't commit time to a new project.

Still, we can discuss protocol & features sometime on Discord...

**Reply by S-Essig** [reply]
Thanks @jonasmalacofilho thats awesome!
Is there an official liquidctl Discord Server? Where can I join? :)

**Reply by jonasmalacofilho** [reply]
Yeah: https://discord.gg/GyCBjQhqCd
